src/sonic-yang-models/yang-models/sonic-telemetry.yang

```diff
@@ -0,0 +1,78 @@
+module sonic-telemetry {
+
+    yang-version 1.1;
+
+    namespace "http://github.com/Azure/sonic-telemetry";
+    prefix telemetry;
+
+    import ietf-inet-types {
+        prefix inet;
+    }
+
+    organization
+        "SONiC";
+
+    contact
+        "SONiC";
+
+    description "TELEMETRY YANG Module for SONiC OS";
+
+    revision 2022-05-13 {
+        description "First Revision";
+    }
+
+    container sonic-telemetry {
+
+        container TELEMETRY {
+
+            description "TELEMETRY TABLE part of config_db.json";
+
+            container certs {
+
+                leaf ca_crt {
+                    type string {
+                        pattern '(/[a-zA-Z0-9_-]+)*/dsmsroot.cer';
```

Comment:
>dsmsroot.cer [](http://example.com/codeflow?start=52&length=12)

Is this pattern enforced by sonic-telemetry implementation? If not, we can relax.

Reply:
According to:

https://github.com/Azure/sonic-mgmt/blob/a51f9fa3125ebca8a60f8e55c7f46eaf8a0e5754/ansible/group_vars/all/telemetry_certs.yml,

All telemetry certs will have the exact path listed. Maybe it makes sense to enforce the entire path instead of just the file name and extension.

Reply:
The file name must be dsmsroot.cer? Can we use other name like abc.cer?

Reply:
Changed pattern to not include filename in the case that file name changes in the future.